code/Data exploration.R

```diff
@@ -53,8 +56,37 @@ data <- data %>% arrange(DATETIME);
 ##fill in missing temperature (down fill)
 data <- data %>% fill(TEMPERATURE)
 
-##convert DATETIME to date
-data$DATETIME = ymd_hms(data$DATETIME);
+
+##create table with period of initial forecast
+initial_forecast <- forecastdemand_nsw %>% 
+  filter(PERIODID==1) %>% 
+  select(DATETIME,
+         FORECASTDEMAND)%>%
+  rename('INITIAL_FORECAST'='FORECASTDEMAND')
+
+##create table with period of final forecast
+final_forecast_period <- forecastdemand_nsw %>% 
```

Comment:
I'm not too sure what's going on here, is this the same as:

```
forecastdemand_nsw %>% 
  group_by(DATETIME) %>% 
  summarise(PERIODID = max(PERIODID))
```

Reply:
it is, and your code is cleaner 👍